src/tools/technology/generate-vars

```diff
@@ -38,12 +38,22 @@ for library_desc in techfile["libraries"]:
 		of.write(" {0}/{1}".format(tech_home, library_desc["nldm library file"]))
 of.write("\n")
 
+# Verilog files required for simulation only.
 of.write("TECHNOLOGY_VERILOG_FILES =")
```

Comment:
I think we should change this name to be explicit i.e. TECHNOLOGY_VERILOG_SIMULATION_FILES